example_panther_config.yml

```diff
@@ -1,6 +1,5 @@
 # Example configuration file for panther_analysis_tool.
 # Copy this file into the root of your test repository and name it .panther_settings.yml
-# Note that options in this file override options passed on the command line
```

Comment:
Can we just update the note to be accurate?

Reply:
Okay, how about something like "Note that options in this file will be overwritten by options passed on the command line"?

Reply:
Nit: s/overwritten/overridden but otherwise LGTM